package.json

```diff
@@ -22,7 +22,20 @@
     "dev": "electron-webpack dev",
     "compile": "electron-webpack",
     "dist": "yarn compile && electron-builder --win --x64",
-    "publish": "yarn compile && electron-builder --win --x64 --publish always"
+    "dist32": "yarn compile && electron-builder --win --ia32",
```

Comment:
I would just add `--ia32` to the regular `"dist"`, no?

Reply:
That would work too, I just figured that if someone was using `dist` instead of `publish` that they'd likely be testing for their own system, and would not want the longer build time caused by building for both architectures. It's an easy enough change though if most people would rather have one longer `dist` command.

Reply:
Or maybe add a `dist32` and `dist64` scripts?

Reply:
Good idea! `dist32` already existed, so I just added `dist64` to match - and now `dist` targets both platforms, similar to `publish`.